src/ocm/components/hosts/UpdateDay2ApiVipDialogToggle.tsx

```diff
@@ -7,5 +7,12 @@ export const UpdateDay2ApiVipDialogToggle: React.FC = () => {
     UpdateDay2ApiVipDialog: { open },
   } = useModalDialogsContext();
 
-  return <AlertActionLink onClick={() => open()}>Update cluster with API IP</AlertActionLink>;
+  return (
+    <>
+      Check your DNS and network configuration, or{' '}
+      <AlertActionLink onClick={() => open()} isInline>
+        set the IP or domain used to reach the cluster
+      </AlertActionLink>
```

Comment:
I think there should be a dot at the end of this sentence as well.

Reply:
Why we need a dot?
![image](https://user-images.githubusercontent.com/11390125/186633245-6a0509d4-d104-42ba-875a-c3a35aaa5f35.png)

Reply:
This is the part this code corresponds to, isn't it? I think there should be a dot at the end of the second sentence after the link.

![image](https://user-images.githubusercontent.com/87187179/186633665-2217eaa2-5377-4791-84ec-62f6a9665365.png)

Reply:
true. Changed it.